apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -249,7 +256,7 @@ protected void run(
       int submitOperationPreSlaGracePeriodInMinutes,
       int statusOperationPreSlaGracePeriodInMinutes) {
     long startTimeMillis = System.currentTimeMillis();
-    METER.counterBuilder("scheduler_start_count").build().add(1);
+    otelEmitter.count(METRICS_SCOPE, "scheduler_start_count", 1, null);
```

Comment:
Can you also pass the attribute type here so that job type is also emitted like it is done in reportMetrics method?